mass/main.py

```diff
@@ -59,6 +93,8 @@ async def consume_events(run_forever: bool = True):
 
     config = Config()
 
+    collection_init_and_index_creation(config)
```

Comment:
Index is only required for search over API, when there is no API we don't need it. So I think indexing here is redundant, it should be enough to have it only run_rest()